includes/html/api_functions.inc.php

```diff
@@ -1382,13 +1382,21 @@ function list_bills(\Illuminate\Http\Request $request)
         if (strtolower($bill['bill_type']) == "cdr") {
             $allowed = format_si($bill['bill_cdr'])."bps";
             $used    = format_si($rate_data['rate_95th'])."bps";
-            $percent = round(($rate_data['rate_95th'] / $bill['bill_cdr']) * 100, 2);
+            if ($bill['bill_cdr'] > 0) {
+                $percent = round(($rate_data['rate_95th'] / $bill['bill_cdr']) * 100, 2);
+            } else {
+                $percent = "-";
```

Comment:
wouldn't it not be better to replace this `$percent = "-";` with `$percent = null;` ?

Reply:
I should have. But I saw the overuse variable is set to "-" when it cannot be calculated so it made me doubt myself. (Overuse is a string with a number, percentage is a float)